src/main/java/io/github/fileanalysissuite/adaptersdk/impls/jaxrs/internal/utils/InstantFunctions.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2022 Micro Focus or one of its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.github.fileanalysissuite.adaptersdk.impls.jaxrs.internal.utils;
+
+import java.time.Instant;
+
+public final class InstantFunctions
```

Comment:
This is a utility functions class.  As such it should have a private empty default constructor.

Reply:
Have added private constructor so class cannot be instantiated from public constructor added by java